### ISSUE TITLE
Create a mailer to remind maintainers to enable MFA one week before MFA policy enforcement

### ISSUE THREAD

**Issue by bettymakes** [rubygems/rubygems.org#3164]
Contributes to [MFA RFC - Phase 3](https://github.com/rubygems/rfcs/pull/36/files#diff-3d5cc3acc06fe7e9150fdbfc43399c5ad42572c122187774bfc3a4857df524f1R69-R84)

---

Per the [MFA announcement](https://blog.rubygems.org/2022/06/13/making-packages-more-secure.html), we will be sending out a mailer to remind users to enable MFA, one week before the MFA enforcement policy goes into effect.

To ensure the mailer is relevant and actionable, this reminder mailer will target the cohort of users who are most impacted by the enforcement of the [MFA policy](https://github.com/rubygems/rfcs/pull/36/files#diff-3d5cc3acc06fe7e9150fdbfc43399c5ad42572c122187774bfc3a4857df524f1R69-R84). 
The specific cohort of users we should target are:
* Maintainers of gems that have 180M+ downloads
* The MFA level on their accounts are set to either `disabled` or `ui_only`
  * In other words, they do not have MFA enabled, or they have weak MFA enabled  
  * If the MFA level is `disabled`, the CTA is to enable MFA
  * If the MFA level is `ui_only`, the CTA is to strengthen their MFA level

The purpose of the email is to compel maintainers to enable MFA before August 15, 2022. 
Based on the latest data (as of August 3), the number of accounts that fall under this cohort of users are:
* `54` accounts with `disabled`
* `6` accounts with `ui_only`
